firestore-send-email/functions/src/logs.ts

```diff
@@ -81,6 +81,14 @@ export function missingUids(uids: string[]) {
   );
 }
 
+export function noPartialAttachmentSupport() {
+  logger.log("partial attachments are not handled and will be ignored");
```

Comment:
```suggestion
  logger.warn("partial attachments are not handled and will be ignored");
```

Let's `warn` here, since it means someone misunderstood how to use the extension.